packages/fuselage/src/components/Sidebar/Sidebar.styles.scss

```diff
@@ -99,7 +99,7 @@ $sidebar-banner-background-default: theme(
 );
 $sidebar-banner-color-default: theme(
   'sidebar-banner-color-default',
-  colors.font(default)
+  colors.font(pure-white)
```

Comment:
@juliajforesti What would be the appropriate way of changing the font color?

Reply:
what do you mean by appropriate way?
the "way" is correct, but the token `pure-white` (meaning **always white**, on dark and light theme) cannot be applied on this surface (`surface-neutral`), because on light mode it has no contrast:
<img width="473" alt="screen" src="https://github.com/RocketChat/fuselage/assets/60678893/fcd8edb0-309f-4e0d-9130-38682066ff25">

was this change requested by the design team? if so, I suggest bringing this issue back to discussion with them, reminding that we have dark and light theme to be checked when we change colors :)

Reply:
@aleksandernsilva font `titles-labels` is darker on light-mode and "whiter", maybe that serves the purpose?